packages/app/src/app/overmind/namespaces/editor/state.ts

```diff
@@ -158,7 +168,14 @@ export const state: State = {
 
       // If this is a single directory with no children
       if (!Object.keys(modulesObject).some(p => dirname(p) === path)) {
-        modulesObject[path] = { ...d, type: 'directory' };
+        // We need to track title and directoryShortid due to
+        // that affectings its path
+        modulesObject[path] = {
```

Comment:
I believe that BrowserFS also uses `updatedAt` and `createdAt` to set the `mtime` and created time. `mtime` is used by the file watcher of VSCode.